privacyidea/static/templates/menu.html

```diff
@@ -31,70 +31,63 @@
         <div class="navbar-collapse collapse myMainNavbar">
             <ul class="nav navbar-nav" ng-hide="token_wizard">
 
-                <li ng-class="{active: $state.includes('dashboard')}"
+                <li ui-sref-active="active"
```

Comment:
Why is that removed? The dashboard should not always be active?
We use the "$state.includes" to mark the top menu as active or not.